demo-app/src/main/java/com/jfcbuilder/demo/JFreeChartBuilderDemo.java

```diff
@@ -234,70 +236,147 @@ private static JFreeChart multiPlotMinuteTimeSeries() {
-      .series(XYTimeSeriesBuilder.get().name("MA(200)").data(sma200).color(Color.RED).style(SOLID_LINE))
-      .annotation(XYArrowBuilder.get().x(stockEventDate).y(stockEventPrice).angle(270.0).color(DARK_GREEN)
-        .textAlign(TextAnchor.BOTTOM_CENTER).text(String.format("%.2f", stockEventPrice)))
-      .line(LineBuilder.get().horizontal().at(resistanceLevel).color(Color.LIGHT_GRAY).style(SOLID_LINE)))
-
-    .xyPlot(VolumeXYPlotBuilder.get().yAxisName("Volume").yTickFormat(volNumFormat).gridLines()
+      .series(XYTimeSeriesBuilder.get().name("MA(200)").data(sma200).color(Color.RED).style(SOLID_LINE));
+    
+    OhlcPlotBuilder volumePlot = VolumeXYPlotBuilder.get()
+      .yAxisName("Volume").yTickFormat(volNumFormat).gridLines()
       .series(VolumeXYTimeSeriesBuilder.get().ohlcv(dohlcv).upColor(Color.DARK_GRAY).downColor(Color.RED))
-      .series(XYTimeSeriesBuilder.get().name("MA(90)").data(volSma90).color(Color.BLUE).style(SOLID_LINE))
-      .annotation(XYArrowBuilder.get().x(stockEventDate).y(stockEventVolume).angle(270.0).color(DARK_GREEN)
-        .textAlign(TextAnchor.BOTTOM_CENTER).text(String.format("%.0f", stockEventVolume)))
-      .line(LineBuilder.get().horizontal().at(volumeLine).color(DARK_GREEN).style(SOLID_LINE)))
-
-    .xyPlot(XYTimeSeriesPlotBuilder.get().yAxisRange(0.0, 100.0).yAxisTickSize(50.0).gridLines()
+      .series(XYTimeSeriesBuilder.get().name("MA(90)").data(volSma90).color(Color.BLUE).style(SOLID_LINE));
+    
+    XYTimeSeriesPlotBuilder tsPlot = XYTimeSeriesPlotBuilder.get()
+      .yAxisRange(0.0, 100.0).yAxisTickSize(50.0).gridLines()
       .yAxisName("Stochastics(" + K + ", " + D + ")")
       .series(XYTimeSeriesBuilder.get().data(stoch.getPctK()).color(Color.RED).style(SOLID_LINE))
       .series(XYTimeSeriesBuilder.get().data(stoch.getPctD()).color(Color.BLUE).style(SOLID_LINE))
       .line(LineBuilder.get().horizontal().at(80.0).color(Color.BLACK).style(SOLID_LINE))
       .line(LineBuilder.get().horizontal().at(50.0).color(Color.BLUE).style(SOLID_LINE))
-      .line(LineBuilder.get().horizontal().at(20.0).color(Color.BLACK).style(SOLID_LINE)));
+      .line(LineBuilder.get().horizontal().at(20.0).color(Color.BLACK).style(SOLID_LINE));
+    
+    if (annotate) {
+
+      // Points ("p") of interest in the plots
+      int lookback = 10;
+      int p1Index = timeArray.length - 1 - lookback;
+      long p1Date = timeArray[p1Index];
+      double p1Price = dohlcv.highs()[p1Index];
+      double p1Volume = dohlcv.volumes()[p1Index];
+      double p1Stoch = stoch.getPctK()[p1Index];
+      
+      lookback = 5;
+      int p2Index = timeArray.length - 1 - lookback;
+      long p2Date = timeArray[p2Index];
+      double p2Price = dohlcv.highs()[p2Index];
+      double p2Volume = dohlcv.volumes()[p2Index];
+      double p2Stoch = stoch.getPctK()[p2Index];
+      
+      double resistanceLevel = dohlcv.closes()[0];
+      double volumeLine = dohlcv.volumes()[0];
+      
+      ohlcPlot
```

Comment:
Adding to these three plots is really the interactive test for all annotations. They are now added to each builder type (OHLC, Volume, XY Time Series).